tests/test_wheel.py

```diff
@@ -42,3 +42,13 @@ def test_choose_returns_random_bin_object_with_seed(self):
         randomly_selected_bin_object = self.wheel.choose()
 
         self.assertIn(self.oc1, randomly_selected_bin_object)
+
+    def test_get_outcome_returns_outcome_object(self):
+        self.wheel.all_outcomes[self.oc1.name] = self.oc1
```

Comment:
better invoke `addOutcome` instead of directly updating `all_outcomes`. This is because in source code `all_outcomes` is updated by `addOutcome`, not directly.

Reply:
Done. Thanks.